av1/av1codecconfigurationrecord.go

```diff
@@ -0,0 +1,99 @@
+package av1
+
+import (
+	"io"
+
+	"github.com/Eyevinn/mp4ff/bits"
+)
+
+// DecConfRec - AV1CodecConfigurationRecord
+// Specified in https://github.com/AOMediaCodec/av1-isobmff/releases/tag/v1.2.0
+type DecConfRec struct {
+	Marker                           byte
```

Comment:
Since Marker must be 1, I think it is better to parse the value and return an error if it is not 1.